src/gateway/bridge/mod.rs

```diff
@@ -72,7 +72,7 @@ pub enum ShardQueuerMessage {
     /// Message to set the shard total.
     SetShardTotal(NonZeroU16),
     /// Message to start a shard.
-    Start(ShardId),
+    Start(ShardId, bool),
```

Comment:
This should be converted to a struct style variant, so the bool can be named.

Reply:
Should I change `ShutdownShard` into a struct variant as well, since it has more than one parameter?

Reply:
Yes